.github/workflows/cloud_continuous_integration.yml

```diff
@@ -7,7 +7,7 @@ on:
       - '.github/workflows/cloud_continuous_integration.yml'
       - '.github/workflow_scripts/test_cloud.sh'
   pull_request_target:
-    types: [labeled, synchronize]
```

Comment:
Why is this needed?

Reply:
I noticed the cloud CI won't be triggered when the PR get opened and will only be triggered with following commits